### ISSUE TITLE
Missing option to hide the row id column

### ISSUE THREAD

**Issue by v217** [gristlabs/grist-core#220]
For the end user, it's often not necessary to see the row id column, wasting valuable screen space. An option to hide this column would make this software even better!
Thanks!

**Comment by alexmojaki** [comment]
Are you referring to these numbers on the left?

![Screenshot from 2022-08-01 16-08-09](https://user-images.githubusercontent.com/3627481/182166418-a9e1943b-db5e-4e6d-ae10-d9bb50deba22.png)

**Comment by v217** [comment]
Yes, not sure, if this is the ID-column, sorry for the confusion, but a hiding option would be great!

**Comment by hugo-laestander** [comment]
I agree with adding an option to remove the ID column / numbering (column most left) from showing. For my use case I have no need to see the ID and in my case it's both visually unpleasing and cluttering the screen. Hope this option can be added, thank you :)

**Comment by dsagal** [comment]
I have a PR implementing an approach with a choice between hiding row numbers entirely or showing the internal Row ID. Preview here: https://grist-gristlabs-grist-core-row-numbers.fly.dev/46ZHFZ2Fdq8T/Lightweight-CRM/m/fork/p/1

<img width="2048" height="1280" alt="Image" src="https://github.com/user-attachments/assets/d439899c-6309-4d48-ac4f-1c696553994f" />

Feedback welcome!
